release.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e
+
+CRATES="pest pest_meta pest_vm pest_generator pest_derive pest_grammars"
+
+get_local_version() {
+  cargo metadata --format-version 1 | jq -r '.packages[]|select(.name == "'"${1}"'" and ((.manifest_path|contains("registry"))|not))|.version'
+}
+
+check_version_online() {
+  curl -s "https://crates.io/api/v1/crates/${1}" | jq -r '.versions[]|select(.num == "'"${2}"'").updated_at'
+}
+
+get_manifest_path() {
+  cargo metadata --format-version 1 | jq -r '.packages[]|select(.name == "'"${1}"'" and ((.manifest_path|contains("registry"))|not))|.manifest_path'
+}
+
+publish() {
+  echo "Publishing crate $1..."
+  cargo publish --manifest-path "$(get_manifest_path "${1}")" --allow-dirty --all-features
+  echo ""
+}
+
+wait_until_available() {
+  echo "Waiting for crate ${1} to become available via crates.io..."
+  for retry in {1..5}; do
+    sleep 5
+    ONLINE_DATE="$(check_version_online "${1}" "${2}")"
+    if [ -n "${ONLINE_DATE}" ]; then
+      echo "Crate ${crate} is now available online"
+      break
```

Comment:
If this line is hit, it will still wait 10 seconds at the end?

Reply:
yes